test/sharness/t0040-add-and-cat.sh

```diff
@@ -156,6 +156,15 @@ test_add_cat_file() {
     test_cmp expected actual
   '
 
+  test_expect_success "ipfs add --chunker rabin-36-512-1024 succeeds" '
+    ipfs add --chunker rabin-36-512-1024 mountdir/hello.txt >actual
```

Comment:
Given that we're not actually *checking* these results, we should probably just run this command as `ipfs add -Q` (quiet) and *not* redirect to `actual`. We redirect to a file when comparing output with some expected output.